src/status_im/subs/chat/messages.cljs

```diff
@@ -27,6 +27,12 @@
     vals
     (sort-by :pinned-at <))))
 
+(re-frame/reg-sub
+ :chats/pin-modal
+ :<- [:messages/pin-modal]
+ (fn [pin-modal [_ chat-id]]
+   (get-in pin-modal [chat-id])))
```

Comment:
Since it's only one key, Can we substitute this `get-in` with `get` ?
`(get pin-modal chat-id)`
or
`(chat-id pin-modal)`

Reply:
Thanks @ibrkhalil. Fixed it.